### DISCUSSION TITLE
Specific key mapping not working on Mac

### DISCUSSION THREAD

**Discussion by humblehacker** [neovide/neovide#2163] [Q&A] [answered]
Does anyone have any idea why this keymapping (⇧⌘o) is not working (on Mac)?

```
vim.keymap.set('n', '<S-D-o>', '‹cmd>Telescope find_files<cr>', { desc = 'Find files' })
```

When I try to use it, it's as if shift and command are ignored and I just typed `o`.

This keymap works when I run `nvim` from the [Kitty terminal](https://github.com/kovidgoyal/kitty) on macOS.

**Comment by fredizzimo** [comment, marked as answer]
It should be fixed on main with this PR https://github.com/neovide/neovide/pull/2018 and this https://github.com/neovide/neovide/pull/2046  will fix some further combinations

**Comment by humblehacker** [comment]
Thanks! I should have done a better search. Hopefully there will be a new release soon.
